### PR TITLE
Multiple commits subject for review

### PR DESCRIPTION
**Pull request by hackatom**
These chunk of commits were merged, unfortunately, making it difficult for @joncamfield to review. We're making a run on this, and the end goal is to make next commits more on a granular level.

### CODE REVIEW

**Comment by joncamfield** [comment]
So far, we have reviewed and it's ready for merging and/or has issues filed to resolve remaining needs:

* Assessment plan changes
* DNS enumeration 
* VOIP assessment

Still need to review:

* Network Mapping
* router attacks
* web vuln assessment

File: en/exercises/network_mapping/considerations.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Considerations:
+
+### Shunning:
+	* If the target organization has IDS/IPS or Firewall in place, it is important to deal with it before conducting the mapping activity. In most cases, these activities are being blocked or either denied by these security technologies in placed. The reason behind why we should allow all traffic from our source machine is that to make sure that we are able to reach the target network to rule out the possibility of attackers being able to bypass these security technologies in parallel scans.
```
Comment (dlshad) [inline en/exercises/network_mapping/considerations.md]: It's important to explain where IDS/IPS are located, and where the auditor is located, outside or inside the network. I've seen few organizations using their IDS/IPS actively inside their LANs.

File: en/exercises/voip_assessment/summary.md [hunk 2]
```diff
@@ -0,0 +1 @@
+VoIP technologies are commonly used nowadays as it provides an alternate flexible way of communication. With its numerous benefits, from toll-bypass, unified voice and data trunking and universally accessible voice-mail and fax-mail services, VoIP services has indeed come into its place as one of the most used communication services today. However, with the rise of cyber attacks, and the reality that any device that connects online can be a potential risk for attacks, VoIP has been on of the favorite target of spam, Interruptions, Voice phishing Hacking and privacy loss.
```
Comment (dlshad) [inline en/exercises/voip_assessment/summary.md]: VOIP Services are commonly used nowadays as they provide a flexible alternative for communication. With numerous services and benefits like toll-bypass, unified voice and data trunking, global accessibility, voice-mail and fax mail services, VoIP is essential to the work of any organizations. On the other hand, VoIP has been one of the most targeted infrastructure by SPAM, Interruptions, hacking and privacy loss.

**Comment by joncamfield** [comment]
Removed the network mapping exercise per https://github.com/SAFETAG/SAFETAG/issues/324 ; let's discuss how to better re-work this in that issue

File: en/references/voip_security.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+* *Guide:* ["VoIP Security Checklist"](https://www.computerworld.com/article/2567181/networking/the-voip-security-checklist.html) (ComputerWorld)
+* *Overview:* ["The Vulnerability of VoIP"](https://us.norton.com/voip-security-a-primer/article) (Symantec)
+* *Research:* ["Researchers find VoIP phones vulnerable to Simple Cyber attacks"](https://securityintelligence.com/news/researchers-find-voip-phones-vulnerable-to-simple-cyberattacks) (Security Intelligence)
+* *Tool:* ["Vsaudit (Eurialo)"](https://github.com/eurialo/vsaudit) (Eurialo)
```
Comment (dlshad) [inline en/references/voip_security.md]: Reviewed this, it's great! can we add this: https://www.symantec.com/connect/articles/two-attacks-against-voip
which covers (Registration Hijacking) and (Eavesdropping) in details

File: en/references/wireless_access_guides.md [hunk 4]
```diff
@@ -32,3 +32,6 @@
   * *Guide:* ["Cracking WPA/WPA2 with oclHashcat"](http://hashcat.net/wiki/doku.php?id=cracking_wpawpa2) (Hashcat wiki)
 
   * *Guide:* ["Wireless Network Review"](http://www.amanhardikar.com/mindmaps/Wireless.html) (amanhardikar.com)
+
+  * *Guide:* ["Router HAcking"](http://www.amanhardikar.com/mindmaps/Wireless.html) (https://github.com/justinbeatz)
```
Comment (dlshad) [inline en/references/wireless_access_guides.md]: @hackatom I see the same link twice (http://www.amanhardikar.com/mindmaps/Wireless.html)

Comment (joncamfield) [reply en/references/wireless_access_guides.md]: And I think the Justinbeatz link probably needs to point to a specific repo, I'm guessing https://github.com/justinbeatz/Armory ?

Comment (dlshad) [reply en/references/wireless_access_guides.md]: or, https://github.com/justinbeatz/Stockpile

File: en/exercises/web_vulnerability_assessment/approach.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+* Understand the current infrastructure the website is using on the level of the hosting provider, location, Operating system
+* Identify the public IP address of the server you will be auditing as you will see in somecases, websites are using proxies or DDoS mitigation services that mask the real IP address of a server
+* In the case of shared hosting, identify the hosting service and the current package 
+* Identify the web server, applications in use & plugins, themes, security protocols in place and users' session management
+* Identify misconfigurations, sensitive information publicly available, metadata embedded within the web application
+* Look for forgotten or insecure support applications like /phpmyadmin   
+* Run automated vulnerability scans against websites hosted by the organization to identify "low-hanging fruits" especially in the case of auditing an opensource and common content management system or other web applications  
+* Perform manual vulenrability asessment and testing to identify server misconfigurations, web sessions, tokens etc.
```
Comment (dlshad) [inline en/exercises/web_vulnerability_assessment/approach.md]: I added some extra lines!

File: en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+**Introduction**
+
+According to Burp's official [documentation](https://support.portswigger.net/customer/portal/articles/1783101-how-to-use-burp-suite), Burp Suite is an integrated platform for performing security testing of web applications. It is **not a point-and-click** tool, but is designed to be used by hands-on testers to support the testing process. With a little bit of effort, anyone can start using the core features of Burp to test the security of their applications. Some of Burp's more advanced features will take further learning and experience to master. All of this investment is hugely worth it - Burp's user-driven workflow is by the far the most effective way to perform web security testing, and will take you way beyond the capabilities of any conventional point-and-click scanner. Burp is intuitive and user-friendly, and the best way to start learning is by doing. These steps will get you started with running Burp and using its basic features. You can then read on deeper into the documentation to become more proficient in using this supremely powerful tool.
```
Comment (dlshad) [inline en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md]: Adding reference to the original documentation

Comment (joncamfield) [reply en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md]: It looks like a lot of the burp walkthrough is from their own documentation - in cases where the tool actually is well documented (and appears to keep that documentation more or less up to date with the tool), it's best to just point auditors there for the specific tool-level needs, and instead provide a more "meta" guide -- see the exercises covering nmap, wireshark, and recon-ng for some variations adapting to different levels of documentation from the tool providers.

File: en/exercises/web_vulnerability_assessment/online_tools.md [hunk 7]
```diff
@@ -1,9 +1,10 @@
-# Introduction:
+**Introduction**
 
-Using online tools in assessing web applications is a great way to start your vulnerability assessment. You can profile a web application target by having a clear idea of what comprises a web application, the technologies behind it and from there build particular strategies on conducting your vulnerability assessment. Using online tools are also ways of performing "passive" scans, in which your identity is hidden from the target organization, in cases where there are IDS/IPS, firewalls deployed.
+Using online tools as a starting point in assessing the auditee web application is a good way to start your vulnerability assessment. You can build a profile and a good understanding of the web application by identifying what comprises the web application and technologies behind. From there you can start your next move by putting together different strategies on conducting your vulnerability assessment. For example, after discovering accessable web directories, you can then start looking for forgotten or abandoned files and applications that might contain sensitive information like (Passwords) or an outdated and vulnerable applications.
```
Comment (dlshad) [inline en/exercises/web_vulnerability_assessment/online_tools.md]: Adding extra text, updating the language/Style

File: en/exercises/web_vulnerability_assessment/foca_analyzer.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
+##### **Introduction:**
+
```
Comment (dlshad) [inline en/exercises/web_vulnerability_assessment/foca_analyzer.md]: Same applies here, as most of the language is pulled from is from FOCA's own documentation.
Quoting @joncamfield  comment "in cases where the tool actually is well documented (and appears to keep that documentation more or less up to date with the tool), it's best to just point auditors there for the specific tool-level needs, and instead provide a more "meta" guide -- see the exercises covering nmap, wireshark, and recon-ng for some variations adapting to different levels of documentation from the tool providers"

**Comment by joncamfield** [comment]
We're close! @hackatom please look at the Burp and FOCA comments.  When you're updating the FOCA guide, please also reformat to have no headers in the file at all to work with the rest of the formatting.

File: en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md (hunk 6 above)
Comment (hackatom) [reply en/exercises/web_vulnerability_assessment/manual_testing_with_burp.md]: Got it @joncamfield!

File: en/exercises/web_vulnerability_assessment/foca_analyzer.md (hunk 8 above)
Comment (hackatom) [reply en/exercises/web_vulnerability_assessment/foca_analyzer.md]: Will do @dlshad!

File: en/exercises/web_vulnerability_assessment/approach.md (hunk 5 above)
Comment (hackatom) [reply en/exercises/web_vulnerability_assessment/approach.md]: Thanks @dlshad!

**Comment by joncamfield** [comment]
Thanks for the foca edits tom - as I dug into it more I realized it fit better in the automated recon activity, so I've moved it there as a variant alongside recon-ng.  I might merge the online tools variant and the existing cms_version activity into a website footprinting activity that both recon and vuln scanning can refer to, and the more in depth web vuln scan activities here would tie only to vuln scanning.

**Comment by hackatom** [comment]
Hey @joncamfield, I've made some minor changes on the dns_enum.md. with the DNS history research, I this is the best place we can add it. I'm dropping the dns_zone_transfer, and will create the dns_history_research or dns_history_analysis if you'll approve.

**Comment by joncamfield** [comment]
OK, I'm working to review and merge the remaining changes.  You took the process mapping in a different but useful direction, I have moved that work into the preparation and related exercises.  I still want to review the following before pressing go on the merge:

* DNS enumeration -- merge individual  files into variants and link in to instructions (@hackatom -- you could help here by copying the file and include approach we used in web_vulnerability_assessment, to put variant_ before the custom instruction files and using INCLUDE statements in the instructions)
* make sure new modules / exercises / references are linked in - voip, dns enum, web_footprint, vuln_scanning; references: voip, sensitive data, incident handling
* review vuln scanning method and exercise setup

**Comment by joncamfield** [comment]
OK, I am merging this beast! @hackatom - can you revisit the VOIP exercise and fill it out a bit further? I put some basic content in there, but I feel instructions for (a) nmapping voip equipment (b) finding out the provider and/or technologies in use (c) checking patch/update status and (d) does the org care about intra-network call encryption (obviously once a voip call connects to a "normal" phone, it is unencrypted), plus some recommendations to address key problems would finish that exercise for the time being.

**Pull request merged**